Homework11/Links.cs

```diff
@@ -0,0 +1,98 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Interactions;
+using Homework11.Configs;
+using OpenQA.Selenium.Support.UI;
+
+namespace Homework11
+{
+    public class Links
+    {
+        public class ButtonTests
+        {
+            private IWebDriver _driver;
+            private WebDriverWait _driverWait;
+
+            [SetUp]
+            public void SetUp()
+            {
+                _driver = new ChromeDriver();
+                _driver.Manage().Window.Maximize();
+                _driver.Navigate().GoToUrl(Constants.linksPage);
+                _driverWait = new WebDriverWait(_driver, TimeSpan.FromSeconds(30));
+            }
+
+            //Test1: Check links that open new tab.
+            [Test]
+            public void VerifyLinksOpenNewTabs()
+            {
+                var homeLink = _driver.FindElement(By.XPath("//a[@id='simpleLink']"));
+                homeLink.Click();
+                Assert.AreEqual(Constants.homePage, GetNewWindowUrl());
+
+                SwitchToOriginalWindow();
+                var homeDynamicLink = _driver.FindElement(By.XPath("//a[@id='dynamicLink']"));
+                homeDynamicLink.Click();
+                Assert.AreEqual(Constants.homePage, GetNewWindowUrl());
+            }
+
+            //Test2: Check links that simulate work with API.
+            [Test]
+            public void VerifyLinksSendApiCall()
+            {
+                var createdLink = _driver.FindElement(By.XPath("//a[@id='created']"));
+                createdLink.Click();
+                Assert.AreEqual(ExpectedConstants.expectedMessageCreated, GetResponseMessage());
+
+                var noContentLink = _driver.FindElement(By.XPath("//a[@id='no-content']"));
+                noContentLink.Click();
+                _driverWait.Until(_driver => GetResponseMessage().Equals(ExpectedConstants.expectedMessageNoContent));
+                Assert.AreEqual(ExpectedConstants.expectedMessageNoContent, GetResponseMessage());
```

Comment:
You already wait expected message in "waiter" - it's a kind of "assert". So there are 2 possible approaches
1. Leave only the first part - wait for expected messages in WebDriverWait
2. Get displayed (not exactly expected) messages form WebDriverWait and compare in NUnit Asserts.
I prefer the second approach - in case test fails you will see "readable" error in output